driver/ppm_compat_unistd_32.h

```diff
@@ -357,11 +357,14 @@
 #define __NR_ia32_renameat2 349
 #define __NR_ia32_userfaultfd 350
 #define __NR_ia32_openat2 351
-#define __NR_ia32_execveat 352
+#define __NR_ia32_io_uring_setup 352
```

Comment:
Same as above, so that we reduce diff from master and we do not break already existing things.